src/GitHUD/Git/Command.hs

```diff
@@ -96,3 +99,21 @@ gitCmdCommitTag :: MVar String
 gitCmdCommitTag out = do
   tag <- readProcessWithIgnoreExitCode "git" ["describe", "--exact-match", "--tags"] ""
   putMVar out tag
+
+gitCmdFetch :: String
+            -> IO ()
+gitCmdFetch path = do
+  -- TODO: gbataille - code org
+  isDir <- doesDirectoryExist path
+  stdout <- openFile "/tmp/out" WriteMode
```

Comment:
Considering that you are not using them, why not just ignore them or pipe them to a variable?

Reply:
you mean `stdout` and `stderr`
It's really the setup of the boilerplate. I just did something quick to see the output because at some point I doubted that some test code was even calling it.
You'll notice that gitCmdFetch is not even called anymore. I wrote the 2 parts:
- git fetch
- background "stateful" daemon polling

I need to bind them, replace the hardcoding with config elements, ...